packages/@livestore/common/src/sync/ClientSessionSyncProcessor.ts

```diff
@@ -141,16 +144,18 @@ export const makeClientSessionSyncProcessor = ({
       for (const table of newWriteTables) {
         writeTables.add(table)
       }
-      event.meta.sessionChangeset = sessionChangeset
-      event.meta.materializerHashSession = materializerHash
+      yield* Effect.sync(() => {
```

Comment:
We don't need that `Effect.sync` wrapper here given we don't do anything with it (e.g. error handling etc)